backend/app/crud/config/config.py

```
@@ -0,0 +1,159 @@
+import logging
+from uuid import UUID
+from typing import Tuple
```

Ruff (0.14.5)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)